packages/apps/virtual-machine/val.yaml

```diff
@@ -0,0 +1,22 @@
+    cloudInit: |
+      #cloud-config
+      users:
+        - name: ubuntu
+          lock_passwd: false
+          plain_text_passwd: 'ubuntu123'
```

Comment:
Don't leave insecure defaults, I can just imagine people scanning for VMs running in Cozystack and trying to get in with ubuntu:ubuntu123.